CONTRIBUTING.md

```diff
@@ -0,0 +1,57 @@
+# Contributing to Kanister
+
+## How Can I Contribute?
+  ### * Reporting Bugs
+  Before creating bug reports, please check the list of issues as you might find out that you don't need to create one. Also when creating a bug report, please include as many details as possible as this information may help us resolve the issue faster. To report a bug please submit an issue at https://github.com/kanisterio/kanister/issues
+
+  ### * Suggesting Enhancements
+  You can suggest minor improvements to existing functionality and or a completely new feature. For that  we recommend creating an issue at https://github.com/kanisterio/kanister/issues and labeling it design outlining the feature that you wish to build, and describing how it will be implemented. This gives a chance for review to happen early and ensures no wasted effort occurs.
+
+  ### * Your first code contribution
+  * You can start by looking through `good-first-issue` issues. We follow the [Github Pull Request Model](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests) for all contributions.
+  * For new features, documentation must be included.
+  * Once review has occurred, please rebase your PR down to a single commit. This will ensure a nice clean Git history.
+  
```

Comment:
Can we also add how the commit messages should be(standards that we follow) and give an example of commit message?

Reply:
Not sure If we should add that. I agree that there are some common standards that people should follow, and that applies to this repo as well. But I think we should mention guidelines for committing messages only if there are some mandatory rules that need to be followed. For example, `All commit messages should start with Issuenumber`. For Kanister we don't have such rules.
cc @viveksinghggits

Reply:
I think we should add some guidelines. A set of guidelines where shared in our slack as well. Let's refer them and add a list here.